projects/ad9656_fmc/Makefile

```diff
@@ -0,0 +1,6 @@
+TARGET := ad9656_fmc
```

Comment:
you have a typo in the commit message: "recevied" -> "received"

Reply:
Changed in v7